app/src/ui/lib/text-box.tsx

```diff
@@ -83,7 +83,7 @@ export interface ITextBoxProps {
   /**
    * Callback used when the component loses focus.
    */
-  readonly onBlur?: () => void
+  readonly onBlur?: (value: string) => void
```

Comment:
Can we add just a tiny note about what `value` is here since `value` is such a generic term?